schemas/cloud-gateway-schema.json

```diff
@@ -32,6 +32,83 @@
+                                                                            "patternProperties": {
+                                                                                "^[a-zA-Z0-9_]+$": {
+                                                                                    "type": "object",
+                                                                                    "description": "OAuth2 client registration settings",
+                                                                                    "properties": {
+                                                                                        "issuer": {
+                                                                                            "type": "string",
+                                                                                            "description": "The URI used to discover the configuration information for an OAuth 2.0 or OpenID Connect 1.0 Provider."
+                                                                                        },
+                                                                                        "clientId": {
+                                                                                            "type": "string",
+                                                                                            "description": "The client identifier."
+                                                                                        },
+                                                                                        "clientSecret": {
+                                                                                            "type": "string",
+                                                                                            "description": "The client secret."
+                                                                                        }
+                                                                                    }
+                                                                                }
+                                                                            }
+                                                                        },
+                                                                        "provider": {
+                                                                            "type": "object",
+                                                                            "description": "OAuth2 client authentication provider settings",
+                                                                            "patternProperties": {
+                                                                                "^[a-zA-Z0-9_]+$": {
+                                                                                    "authorizationUrl": {
```

Comment:
I guess here is missing field "properties"

Reply:
thanks for spotting it, done.